treatment-service/config/config.go

```diff
@@ -24,16 +26,17 @@ type Config struct {
 	Port       int      `json:"port" default:"8080" validate:"required"`
 	ProjectIds []string `json:"project_ids" default:""`
 
-	AssignedTreatmentLogger AssignedTreatmentLoggerConfig       `json:"assigned_treatment_logger"`
-	DebugConfig             DebugConfig                         `json:"debug_config" validate:"required,dive"`
-	NewRelicConfig          newrelic.Config                     `json:"new_relic_config"`
-	SentryConfig            sentry.Config                       `json:"sentry_config"`
-	DeploymentConfig        DeploymentConfig                    `json:"deployment_config" validate:"required,dive"`
-	MessageQueueConfig      common_mq_config.MessageQueueConfig `json:"message_queue_config" validate:"required,dive"`
```

Comment:
@rautelaanirudh could you share why this is changed to be a pointer?

Reply:
Since I had removed the validation from above and added a custom validator, hence I changed it to a pointer. Also my understanding/expectation was that if the application is using poller, then the message queue configuration should not necessarily exist.

Reply:
I have changed it back to value instead of a pointer. Thanks for pointing that out.